tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +127,64 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_checks(self):
+        self.git_commit('Test commit')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+        self.git_commit('Revert "Test commit"\n\n'
+                        'Abcdef blablabla commit ' + str(commit_hash) + '\n\n'
+                        'Reason for the revert commit is explained here.\n')
+        self.assertEqual(self.run_uut(),
+                         ['Invalid revert commit.'])
+        self.assert_no_msgs()
+
+        self.git_commit('Test commit1\n\n'
+                        'This is body of test commit1\n')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+
+        self.run_git_command('revert', 'head')
+        self.git_commit('Revert "Another Test commit2"\n\n'
+                        'This reverts commit ' + str(commit_hash) + '\n\n'
+                        'Reason for the revert commit is explained here.\n')
+        self.assertEqual(
+            self.run_uut(),
+            ['Shortlog of revert commit does not match the original commit.'])
+        self.assert_no_msgs()
+
+        self.git_commit('Test commit3')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+        self.run_git_command('revert', 'head')
+
+        self.git_commit('Revert "Test commit3"\n\n'
```

Comment:
This is the default format of a revert commit, you can omit creating this one manually and just use the commit generated in line 155